{
    "status_code": 200,
    "data": {
        "StackEvents": [
            {
                "StackId": "arn:aws:cloudformation:us-west-2:123456789012:stack/ansible-test-on-create-failure-rollback/914046e0-3a0f-11e9-94bf-0a9edf17d014",
                "EventId": "9140bc10-3a0f-11e9-94bf-0a9edf17d014",
                "StackName": "ansible-test-on-create-failure-rollback",
                "LogicalResourceId": "ansible-test-on-create-failure-rollback",
                "PhysicalResourceId": "arn:aws:cloudformation:us-west-2:123456789012:stack/ansible-test-on-create-failure-rollback/914046e0-3a0f-11e9-94bf-0a9edf17d014",
                "ResourceType": "AWS::CloudFormation::Stack",
                "Timestamp": {
                    "__class__": "datetime",
                    "year": 2019,
                    "month": 2,
                    "day": 26,
                    "hour": 21,
                    "minute": 43,
                    "second": 34,
                    "microsecond": 740000
                },
                "ResourceStatus": "CREATE_IN_PROGRESS",
                "ResourceStatusReason": "User Initiated"
            }
        ],
        "ResponseMetadata": {
            "RequestId": "9199b1a7-3a0f-11e9-b938-97983b40cabe",
            "HTTPStatusCode": 200,
            "HTTPHeaders": {
                "x-amzn-requestid": "9199b1a7-3a0f-11e9-b938-97983b40cabe",
                "content-type": "text/xml",
                "content-length": "1161",
                "date": "Tue, 26 Feb 2019 21:43:35 GMT"
            },
            "RetryAttempts": 0
        }
    }
}
